source/agora/consensus/validation/PreImage.d

```diff
@@ -53,8 +53,8 @@ public string isInvalidReason (const ref PreImageInfo new_image,
     if (new_image.enroll_key != prev_image.enroll_key)
         return "The pre-image's enrollment key differs from its descendant";
 
-    if (new_image.distance < prev_image.distance)
-        return "The height of new pre-image is smaller than that of previous one";
+    if (new_image.distance <= prev_image.distance)
+        return "The height of new pre-image is equal to or smaller than that of previous one";
```

Comment:
```suggestion
        return "The height of new pre-image is not greater than that of the previous one";
```

Reply:
Where is the associated unittest for this change?

Reply:
Moved to #1040